cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -612,13 +612,20 @@ func isDeviceEncrypted(name string) (ok bool, encdev string) {
 	return false, ""
 }
 
+// TODO:UC20: move this somewhere appropriate
+var readKernelUUID = func() string {
+	b, err := ioutil.ReadFile("/proc/sys/kernel/random/uuid")
+	if err != nil {
+		panic("can't read kernel uuid")
```

Comment:
nitpick: cannot ...

Reply:
will change in master, do you want this changed in 2.45 as well (with a followup or here)?

Reply:
fixed in master PR